src/includeDataProps.js

```diff
@@ -0,0 +1,11 @@
+export default function (props) {
+  const dataProps = {}
+
+  Object.keys(props).forEach(key => {
+    if (key.indexOf('data-') > -1) {
```

Comment:
Just noticed this will match strings like `foo-data-bar`, which aren't valid data attributes. What do you think of changing the predicate to `/^data-/.test(key)`?